trials/countdown.js

```diff
@@ -6,19 +6,18 @@ const _ = require("lodash");
  *
  * @module
  * @param {Object} options
- * @param {number} options.duration - The trial duration.
- * @param {string} options.text - Optional text for the countdown. (default: "")
- * @param {number} options.time - The number of seconds for the countdown. (default: 10s)
- * @param {boolean} options.responseEndsTrial - True if the trial ends on response, false if the trial waits for the duration. (default: false)
+ * @param {number} options.duration - trial duration in milliseconds. (default: 1000)
+ * @param {string} options.text - (optional) text for the countdown. (default: "")
+ * @param {number} options.time - start number for the countdown. (default: 10)
  */
 
 module.exports = function (options) {
   const defaults = {
+    duration: 1000,
     text: "",
     time: 10,
```

Comment:
can we make this default 3?

Reply:
for the countdown trial, would it make more sense to change the time to 3 instead of duration?

Reply:
yuuup... that's what I meant 🤦